src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,69 @@
+package duke.task;
+
+import duke.exception.DukeException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
```

Comment:
You might want to import the `java.` classes before the `duke.` classes so that your code pass the Style Check later on. 😉